openshift/4.0/templates/backup/README.md

```diff
@@ -1,22 +1,74 @@
 # Database Backup Service
 
-Provides cron based MSSQL backups using plugin developed for [https://github.com/BCDevOps/backup-container])https://github.com/BCDevOps/backup-container).
+Provides cron based MSSQL backups using plugin developed for [https://github.com/BCDevOps/backup-container](https://github.com/BCDevOps/backup-container).
 
 See documentation for that project for complete usage documentation, details on the MSSQL plugin, and all source. The container will automatically backup based on the cron scheduled in the config map. Backups may also be taken manually via backup.sh. Usage is provided in the backup-container repo or via `./backup.sh -h`.
 
 Expected usage within PIMS is primarily to use the cron daily backups. This allows us to restore the database in the event of catastrophic failure or corruption with minimal data loss.
 
-> Database backup does not currently work.
+> We do not presently monitor whether this cronjob fails. This is a high risk area that should be addressed.
```

Comment:
Yeah, I can log a story to figure out getting the rocketchat notifications turned on.

Reply:
*have logged